src/AITS/PersonLookupConfidential.php

```diff
@@ -139,6 +141,9 @@ public function findPerson(string $lookupKey): void
             // Employees will have extra data present, may never use
             $this->employee = $this->json['list'][0]['employee'] ?? [];
 
+            // Employees organization description
+            $this->organizationDescription = $this->json['list'][0]['employee']['organizationDesc'] ?? '';
```

Comment:
@hualiu6 , @jjonesuic - What were to happen if `$this->json['list'][0]['employee']` is undefined? In other words, it's a student?

Generally speaking, I don't think we should define `$this->organizationDescription`. While I understand it's a quick way to get the org for the user, there's a few problems I have with it.

1. The name of the property, `organizationDescription`, implies out of context that the data might be available for students. That isn't going to be the case all the time.
2. It's redundant with `$this->employee`. The org description can be grabbed via `$person->getEmployee()['organizationDesc']`

I'm open to hearing counter arguments for keeping it.

Reply:
That's true, and then we also wouldn't need to include gets for all the elements of the employee array.

Reply:
@thedanpaz @jjonesuic Agree to remove the getOrganizationDescription but keep getEmployee(), this way we are flexible, if we need the unit code in the future.